index.html

```diff
@@ -274,6 +274,15 @@ <h3>What is a Verifiable Credential?</h3>
 using privacy-enhancing technologies, such as zero-knowledge proofs, are also
 provided throughout this document.
         </p>
+
+	<p>
+The word "verifiable" in the terms
+<a>verifiable credential</a> and <a>verifiable presentation</a>
+refers to the characteristic of a <a>credential</a> or <a>presentation</a>
+as being able to be <a>verified</a> by a <a>verifier</a>,
+as defined in this document. Verifiability of a credential does not imply
+that the truth content of <a>claims</a> encoded therein can be evaluated.
```

Comment:
Matching the later change suggestion --
```suggestion
as defined in this document. Verifiability of a credential does not imply
that the truth of <a>claims</a> encoded therein can be evaluated.
```

Reply:
Can we add. "However, the Evidence property can be used by the issuer to help the verifier determine the truth of the <a>claims</a>"

Reply:
@clehner -- please review @TallTed's requested changes as well as @David-Chadwick's requested ones. This PR is ready to go after you do that.

Reply:
@David-Chadwick (@msporny @clehner) -- I've incorporated a paraphrase of your addition to my suggested change. Hopefully this still meets with everyone's approval!

Reply:
Yes fine thanks.

Reply:
Committed the modified suggested change in 4b358f7bb319bc0d8b73cc413a99af44911e8987.

Updated in 1e114de493b03d820c53223baa98554fa4a9650e to add a link to the evidence property definition.

Aside: while searching the Web for the now-removed phrase "truth content", I found it is used in the works of science philosopher Karl Popper; I was not knowingly using the term in that way, but maybe some people will find this interesting.
https://iep.utm.edu/pop-sci/
https://en.wikipedia.org/wiki/Karl_Popper
https://plato.stanford.edu/entries/truthlikeness/